routes/userRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const userController = require('../controllers/userController');
+
+const router = express.Router();
+
+router.get('/kakao/signin', userController.signIn);
+router.post('/kakao/signup', userController.signUp);
```

Comment:
현재는 카카오를 통한 로그인을 구현하도록 기획이 되어 있어 엔드포인트 작성 시 다음과 같이 작성해주신 것으로 보입니다.

다만 서비스를 운영하면서 일반 회원가입, 다른 플랫폼의 로그인 API를 연동하는 경우가 있을 수 있으므로 엔드포인트를 다음과 같이 수정하는 것이 좋을 것 같습니다!
```suggestion
router.get('/signin', userController.signIn);
router.post('/signup', userController.signUp);
```